.github/workflows/ci.yml

```diff
@@ -86,7 +86,7 @@ jobs:
         run: bin/rails db:test:prepare && bin/rspec
 
       - name: Keep screenshots from failed system tests
-        uses: actions/upload-artifact@v4
+        uses: actions/upload-artifact@v5
```

Comment:
TECHNICALLY we should see an upload happen to make sure things are fine but this is just a useful dependabot bump and we should assume 👍